pkg/tools/fit.go

```diff
@@ -0,0 +1,25 @@
+package tools
+
+import (
+	"fmt"
+
+	"github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/fit"
+)
+
+// PrintFit prints the Firmware Interface Table in a tabular human readable form.
+func PrintFit(image []byte) error {
+	fit, err := fit.GetEntries(image)
+	if err != nil {
+		return err
+	}
+	fmt.Println("Firmware Interface Table")
+	fmt.Println("------------------------")
+	fmt.Printf("%-25s | %-20s | %-8s | %-10s | %-15s | %-10s\n", "Type", "Address", "Size", "Version", "Checksum valid", "Checksum")
+	fmt.Printf("-----------------------------------------------------------------------------------------------------\n")
+	for _, entry := range fit {
+		h := entry.GetHeaders()
+
+		fmt.Printf("%-25s | %-20s | %-8d | %-10s | %-15v | %-10d\n", h.Type().String(), h.Address.String(), h.Size.Uint32(), h.Version.String(), h.IsChecksumValid(), h.Checksum)
+	}
+	return nil
+}
```

Comment:
IMHO, it would be better to place this function to `pkg/intel/metadata/fit` (instead of `pkg/tools`). It could be the `String` method of `fit.Table`. Also there's no need to call `GetEntries`: I see you print only headers, so it is enough to call `GetTable` (it will get only headers and won't parse referenced data).